packages/edit-site/src/components/page-patterns/grid-item.js

```diff
@@ -200,6 +200,7 @@ function GridItem( { categoryId, item, ...props } ) {
 					<BlockPreview
 						blocks={ item.blocks }
 						additionalStyles={ additionalStyles }
+						viewportWidth={ item?.viewportWidth }
```

Comment:
Nit: I don't think we need the optional chaining here

Reply:
Indeed 👍